src/modules/account/components/AccountDetails/index.js

```diff
@@ -57,4 +57,4 @@ const AccountDetails = ({ history }) => {
   return renderComponent();
 };
 
-export default compose(withRouter)(AccountDetails);
+export default compose(AccountDetails);
```

Comment:
why do we need compose here? if its not required let's check and remove.

Reply:
This is the only instance where it's necessary to remove it and it's been fixed